### PR TITLE
Feature/fix template errors

### PR DESCRIPTION
**Pull request by perruche**
Fix some errors in the theme template : 

- Correct the way `$post` is retrieve in `front-page.php`
- Suppress an undefined variable in `index.twig`
- Lint twig files 
-  Suppress front-page from template list for is_home() conditionnal

### CODE REVIEW

**Review by titouanmathis — APPROVED** [review]
Great job, thanks!

**Review by lusimeon — APPROVED** [review]

**Comment by perruche** [comment]
I added some minor changes in `functions.php` to avoid warning in the IDE : 
- Correct the deprecated use of `new Twig_Extension_StringLoader`
- Correct the type of the param in `add_to_twig` docblock comment
- Add line breaks to comments to have the same format everywhere

Waiting for your approval to merge

**Review by titouanmathis — APPROVED** [review]
👍

**Review by ptt-homme — APPROVED** [review]
Great job!

**Review by titouanmathis — APPROVED** [review]

File: template/web/wp-content/themes/<%= slug %>/templates/components/pagination.twig
Comment (perruche) [inline template/web/wp-content/themes/<%= slug %>/templates/components/pagination.twig]: Need to double check this file, it's containing some errors

Comment (ptt-homme) [reply template/web/wp-content/themes/<%= slug %>/templates/components/pagination.twig]: @perruche Lets talk about this on monday!

**Review by titouanmathis — APPROVED** [review]
👍

File: template/web/wp-content/themes/<%= slug %>/templates/components/pagination.twig
Comment (titouanmathis) [reply template/web/wp-content/themes/<%= slug %>/templates/components/pagination.twig]: What are the errors? Any idea when it could be fixed? I would like to merge every PR to release a new version before our next WP projects.

Comment (perruche) [reply template/web/wp-content/themes/<%= slug %>/templates/components/pagination.twig]: There is some missing `posts` values in the file:
like : `{% if POSTS.pagination.pages|first and POSTS.pagination.pages|first.current != true %}`

**Pull request merged**
